packages/snap/src/dialogs/address.tsx

```diff
@@ -1,20 +1,80 @@
-import { Box, Text } from '@metamask/snaps-sdk/jsx';
 import { REQUEST_METHODS, DIALOG_TYPES } from '../constants';
+import { Box, Container, Copyable, Heading, Section, Text } from '@metamask/snaps-sdk/jsx';
 
-export const addressPage = async () => (
+const renderParamText = (params) => {
+  switch (params.type) {
+    case 'first_empty':
```

Comment:
question(non-blocking): Does it make sense at this moment in the snaps development to extract those parameter types to constants?

Reply:
I think this ENUM should exist in the RPC handler package, where the rpc method definition is. I created an issue to handle this https://github.com/HathorNetwork/hathor-rpc-lib/issues/61